examples/popmon_dp_loader_example.ipynb

```diff
@@ -0,0 +1,479 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2adec556",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "try:\n",
+    "    sys.path.insert(0, '..')\n",
+    "    import dataprofiler as dp\n",
+    "except ImportError:\n",
+    "    import dataprofiler as dp\n",
+    "import pandas as pd\n",
+    "import popmon  # noqa"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed532ec",
+   "metadata": {},
+   "source": [
+    "## Comparison of Dataloaders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cccbf4cd",
+   "metadata": {},
+   "source": [
+    "First, we have the original pandas dataloading which works for specific file types. \n",
+    "This is good for if the data format is known ahead of time but is less useful for more dynamic cases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96e9ff89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def pm_dataloader(path, time_index):\n",
+    "    # Load pm dataframe (Can only read csvs unless reader option is changed)\n",
+    "    if not time_index is None:\n",
+    "        pm_data = pd.read_csv(path, parse_dates=[time_index])\n",
+    "    else:\n",
+    "        time_index = True\n",
+    "        pm_data = pd.read_csv(path)\n",
+    "    return pm_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16dfbe10",
+   "metadata": {},
+   "source": [
+    "Next, we have the dataprofiler's dataloader. This allows for the dynamic loading of different data formats which is super useful when the data format is not know ahead of time.\n",
+    "This is intended to be an improvement on the dataloader standardly used in pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07481259",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dp_dataloader(path, time_index):\n",
+    "    # Datalaoder from dataprofiler used\n",
+    "    dp_data = dp.Data(path) \n",
+    "\n",
+    "    # Make dataframe from dataprofiler popmon compliant\n",
+    "    if not time_index is None:\n",
+    "        # Mimics parse_dates=[time_index] optional param\n",
+    "        dp_data.data[time_index] = pd.to_datetime(dp_data.data[time_index])\n",
+    "\n",
+    "    return dp_data.data, time_index"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69a8ea9b",
+   "metadata": {},
+   "source": [
+    "## Popmon's usage example using both dataloaders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7cc2c62",
+   "metadata": {},
+   "source": [
+    "To execute this example we are going to need to download data from popmons \"resources\" component so we must import that into our project as well. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e6bb60bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from popmon import resources"
```

Comment:
i would suggest leaving it as `popmon.resources` so that it is apparent that resources comes from `popmon`.